charter.md

```diff
@@ -0,0 +1,34 @@
+# Style Team
+
+## Aims: Evolving the Rust Style over time
+
+1. Selecting styling for new Rust constructs
+1. Evolving the existing style over the course of Rust editions (without breaking backwards compatibility)
+1. Defining mechanisms to evolve the Rust style while taking backwards compatibility into account, such as via Rust editions or similar mechanisms
+
+## Domains
+
+1. [Style Guide]
+1. [T-style team repo](https://github.com/rust-lang/fmt-rfcs/)
+1. [#T-style] and #T-style/private zulip streams
+1. `rust-lang` issues with the `T-style` or `T-style-nominated` labels
+
+## Membership
+
+* Caleb Cartwright ([@calebcartwright](https://github.com/calebcartwright))
+* Jane Losare-Lusby ([@yaahc](https://github.com/yaahc))
+* Josh Triplett ([@joshtriplett](https://github.com/joshtriplett))
+* Michael Goulet ([@compiler-errors](https://github.com/compiler-errors))
+
+## Membership
+
+The Rust style team shall have at least 3 members and at most 8. If the team has fewer than 3 members it shall seek new members as its primary focus.
+
+Members of the style team are nominated by existing members. All existing members of the team must affirmatively agree to the addition of a member, with zero objections; if there is any objection to a nomination, the new member will not be added. In addition, the team lead or another team member will check with the moderation team regarding any person nominated for membership, to provide an avenue for awareness of concerns or red flags.
```

Comment:
shouldn't we have a general policy for this accross the project?, unsure if something like that exists already but I guess don't?

Reply:
I'd love for this to become general policy and I'm hoping this can serve as prior art to show that this approach and framing towards team membership works effectively.

Reply:
Yes, this is what I was guessing :), cool.

Reply:
We should absolutely have general policy for this, we just don't currently have that policy or a good process by which we could adopt such a policy.